plugin/kubernetes/container.go

```diff
@@ -33,9 +34,20 @@ func newContainer(ctx context.Context, client *k8s.Clientset, config *rest.Confi
 	cntnr.ns = ns
 	cntnr.pod = p
 
-	// TODO: Get container startup time from ContainerStatus from the parent Pod
+	// Find when the container was started; set this as the creation time
+	creationTimestamp := time.Now()
```

Comment:
It's ok to leave creationTimestamp unset if it's not in ContainerStatuses. So I'd only update them if you find a match.

You could put
```
cntnr.Attributes().SetAtime(ecs.State.Running.StartedAt.Time)...
```
inside the if block, then break.

Reply:
Good catch. I'll get to that tonight.

Reply:
Fixed.